getting-started/dc2-guide.rst

```diff
@@ -70,30 +70,30 @@ Building good seeing templates
 ------------------------------
 
 The pipeline we will use lives in the ``ap_pipe`` package, and is the camera-specific ``ApTemplate.yaml`` pipeline.
-To see it, either navigate to the `pipeline on GitHub <https://github.com/lsst/ap_pipe/blob/main/pipelines/LsstCamImSim/ApTemplate.yaml>`__ or display the pipeline on via the command line, e.g.,
+To see it, either navigate to the `pipeline on GitHub <https://github.com/lsst/ap_pipe/blob/main/pipelines/LSSTCam-imSim/ApTemplate.yaml>`__ or display the pipeline on via the command line, e.g.,
 
 .. prompt:: bash
 
-   cat $AP_PIPE_DIR/pipelines/LsstCamImSim/ApTemplate.yaml
+   cat $AP_PIPE_DIR/pipelines/LSSTCam-imSim/ApTemplate.yaml
 
 Note that this camera-specific pipeline imports both a camera-specific single-frame processing pipeline (sometimes called "processCcd") and a more generic AP Template building pipeline.
 
 To visualize this pipeline, you may wish to use ``pipetask build``, e.g.,
 
 .. prompt:: bash
 
-   pipetask build -p $AP_PIPE_DIR/pipelines/LsstCamImSim/ApTemplate.yaml --pipeline-dot ApTemplate.dot
+   pipetask build -p $AP_PIPE_DIR/pipelines/LSSTCam-imSim/ApTemplate.yaml --pipeline-dot ApTemplate.dot
    dot ApTemplate.dot -Tpng > ApTemplate.png
 
 Alternately, navigate to `this website that serves visualizations of all the AP and DRP pipelines <https://tigress-web.princeton.edu/~lkelvin/pipelines/current>`__.
-Click through to ``ap_pipe``, then ``LsstCamImSim``, and finally ``ApTemplate`` to find a PDF visualizing all the pipeline inputs, outputs, and intermediate data products.
+Click through to ``ap_pipe``, then ``LSSTCam-imSim``, and finally ``ApTemplate`` to find a PDF visualizing all the pipeline inputs, outputs, and intermediate data products.
```

Comment:
Should we ask @leeskelvin to get rid of the old directories? Otherwise, https://tigress-web.princeton.edu/~lkelvin/pipelines/current/ap_pipe/ and `ap_verify/` will have both the old and new forms, which might confuse users. Sorry for not thinking of this the first time!

Reply:
Files on https://tigress-web.princeton.edu/~lkelvin/pipelines/current/ap_pipe/ are auto-generated each week depending on what's available in the named repos. New names for cameras should be picked up automatically on the next rebuild (scheduled for Friday morning). If not, please do let me know.

Reply:
Yes, but my concern is that the old ones will stick around indefinitely, and new users especially (the target audience of this guide) will have no idea which directories are old and which are new.

Reply:
Older weeklies are stored under: https://tigress-web.princeton.edu/~lkelvin/pipelines/. But the change on this ticket isn't being backported is it? I.e., if a user loads up `w_2024_03`, they'll still see the older names under `ap_pipe`?

Reply:
Pipelines under https://tigress-web.princeton.edu/~lkelvin/pipelines/current will always point to the most recently build weekly.

Reply:
Ah, that's what I was thinking of. Thanks for clarifying.

No, no backporting. If only one version of the directories shows up with any given version, I don't see much room for confusion.